### ISSUE TITLE
[DISCUSSION] Change Google Places to OSMNames or Nominatim

### ISSUE THREAD

**Issue by PoOwAa** [benawad/vsinder#135]
Dear @benawad and Vsinder contributors,

Ben in his latest [video](https://www.youtube.com/watch?v=thqX845NEgg&ab_channel=BenAwad) pointed out that google places costs could be too high to use it. What about change google maps to osm?
[OSMNames](https://osmnames.org/)
[Nominatim](https://nominatim.org/)

The downside of these, that you can not overwhelm their API, so the Vsinder should have its own instance. According to OSMNames [documentation](https://osmnames.org/docs/):

> This file will be indexed on the first run or if index files are missing. The generated indexes for the whole world are large (over 30GBytes) and time-consuming (about 45minutes) to create. But the process is done only once.

So basically it needs a ~30-50Gb container. (The whole world's data file is downloadable about 1.4 Gb)

With this method, there isn't any extra cost, only network traffic if a cloud provider hosts it.

What are your thoughts?

**Comment by PoOwAa** [comment]
A good alternative could be #142

**Comment by unDeadHerbs** [comment]
Seeing as location is currently missing, this has become more important.  A possible crutch for this is #106.

**Comment by rugk** [comment]
Could we keep one issue for this open only, to keep the discussion in one place? So either close https://github.com/benawad/vsinder/issues/142 or this one as a dupe?
